sesman/session.c

```diff
@@ -676,6 +676,12 @@ session_start_fork(int width, int height, int bpp, char *username,
                     /* these are the must have parameters */
                     list_add_item(xserver_params, (tintptr) g_strdup(xserver));
                     list_add_item(xserver_params, (tintptr) g_strdup(screen));
+                    
+                    char xauth_file[256];
+                    g_snprintf(xauth_file, 255, "%s/.Xauthority", g_getenv("HOME"));
+                    list_add_item(xserver_params, (tintptr) g_strdup("-auth"));
+                    list_add_item(xserver_params, (tintptr) g_strdup(xauth_file));
```

Comment:
This should be done for all servers: X11rdp, Xorg and Xvnc. I wonder it it can be done in sesman.ini. The X server should already be running in the user's home directory, so the full path should not be needed. Some setuid X servers reject full path for the log; they could reject full path to `.Xauthority` as well.

Generally, I would prefer the configuration like that to be outside the code for more flexibility. Eventually, all the code for starting backends could become a shell script.